### PR TITLE
Updated python and django versions in Travis & Tox

### PR DESCRIPTION
**Pull request by UsamaSadiq**
**Issue:** [BOM-1528](https://openedx.atlassian.net/browse/BOM-1528)

### Description
- Updated `Travis.yml` to include `python3.8` support.
- Updated `tox.ini` to include `python3.8` and `django3.0`.

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -1,14 +1,16 @@
 [tox]
-envlist = py35-django{22}
+envlist = py{35,38}-django{22,30}
 skipsdist = true
 
 [pytest]
-DJANGO_SETTINGS_MODULE = designer.settings.test
+django_settings_module = designer.settings.test
```
Comment (jmbowman) [inline tox.ini:6]: Why did this change to lower case?  The pytest-django docs still have it as upper case: https://pytest-django.readthedocs.io/en/latest/#quick-start

File: tox.ini [hunk 2]
```diff
@@ -1,14 +1,16 @@
...
 testpaths = designer/apps
 
 [testenv]
-deps =
-    django22:  -r requirements/django.txt
-    -r {toxinidir}/requirements/test.txt
-commands =
-    {posargs:python -Wd -m pytest}
+deps = 
+	django22: Django>=2.2,<2.3
```
Comment (jmbowman) [inline tox.ini:11]: We still want `requirements/django.txt` here, we should fix the script if it changed that.  Looks like the indentation is wrong, too.

File: tox.ini (hunk 1 above)
Comment (UsamaSadiq) [reply tox.ini:6]: I guess, the script changed it to lowercase. I'll change it back to original.

File: tox.ini (hunk 2 above)
Comment (UsamaSadiq) [reply tox.ini:11]: changed to `requirements/django.txt`.

**Pull request merged**